.sandbox/provision/tf-lifecycle/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "launch_template_name" {
+  default = "sandbox-windows-vm"
+}
+
+variable "launch_template_version" {
+  default = "1"
+}
+
+variable "keypair_file" {
+  default = "/run/sandbox/fs/secrets/shared/sandbox-shared.pem"
+}
+
+variable "suspended" {
+  type = bool
+  default = false
+}
+
+variable "availablity_zone" {
+  type = string
+  default = "us-east-2a"
+}
```

Comment:
For all these variables, please add `description`. This is a reference solution, treat it as production code.